src/libs/API/parameters/UpdatePolicyConnectionConfigParams.ts

```diff
@@ -1,11 +1,10 @@
-import type {ValueOf} from 'type-fest';
-import type CONST from '@src/CONST';
+import type {ConnectionName, Connections} from '@src/types/onyx/Policy';
 
-type UpdatePolicyConnectionConfigParams = {
+type UpdatePolicyConnectionConfigParams<TConnectionName extends ConnectionName, TSettingName extends keyof Connections[TConnectionName]['config']> = {
```

Comment:
@srikarparsi, for this command param type, you can refer to [this part of the Web-E code](https://github.com/Expensify/Web-Expensify/blob/213ec750049b3d5d2e803a7a1c9b8a0f77c4d809/lib/Auth.php#L9726-L9739) to check the list of parameters

Reply:
But most of what's happening in this code is combining of different types to form the new type, which is `UpdatePolicyConnectionConfigParams`.

Just type juggling